CHANGELOG.md

```diff
@@ -3,10 +3,23 @@
 NOTE: From this version on the frameworks changed to Dynamic instead of Static library
 
 ### Bug fixes
-- 
+- In implicit authentication flow, completion block of the original request will only get notified once with cancellation of authentication flow, or actual result of the endpoint. Previously, the original request got notified in every error that happened during the authentication flow. [DE311261]
```

Comment:
In the implicit authentication flow, errors are now only reported once during the authentication flow or actual result of the endpoint. Previously, the original request was notified for every error that happened during the authentication flow.